include/extractor/scripting_environment_lua.hpp

```diff
@@ -33,6 +45,8 @@ struct LuaScriptingContext final
     bool has_node_function;
     bool has_way_function;
     bool has_segment_function;
+
+    location_handler_type * location_handler;
```

Comment:
Here is where we store nodes during extraction in the stxxl containers and then serialize out to disk / discard:

- https://github.com/Project-OSRM/osrm-backend/blob/refactor-sliproad-handler/include/extractor/extraction_containers.hpp
- [ExternalMemoryNode](https://github.com/Project-OSRM/osrm-backend/blob/388d84a89ed2eff812e8f68bc4444f76e01abb20/include/extractor/external_memory_node.hpp)
- which is a [QueryNode](https://github.com/Project-OSRM/osrm-backend/blob/master/include/extractor/query_node.hpp)

(Related: let's rename QueryNode to something more semantically correct https://github.com/Project-OSRM/osrm-backend/issues/3238)